gui/elements/qt/_layerList.py

```diff
@@ -0,0 +1,21 @@
+from PyQt5.QtCore import Qt
+from PyQt5.QtWidgets import QWidget, QVBoxLayout, QScrollArea
+
+
+class QtLayerList(QScrollArea):
+    def __init__(self):
+        super().__init__()
+
+        self.setWidgetResizable(True)
+        self.setFixedWidth(300)
+        self.setHorizontalScrollBarPolicy(Qt.ScrollBarAlwaysOff)
+        scrollWidget = QWidget()
+        self.setWidget(scrollWidget)
+        self.layersLayout = QVBoxLayout(scrollWidget)
+        self.layersLayout.addStretch(1)
+
+    def add_layer(self, index, layer):
+        self.layersLayout.insertWidget(index, layer._qt)
```

Comment:
We should check that `_qt` is not `None` first.

Reply:
Can add.